emu/src/arm7tdmi.rs

```diff
@@ -110,6 +111,10 @@ impl Cpu for Arm7tdmi {
     fn registers(&self) -> Vec<u32> {
         self.registers.to_vec()
     }
+
+    fn get_memory(&self) -> Ref<'_, InternalMemory> {
```

Comment:
You can return `Rc<RefCell<InternalMemory>>` instead with `self.memory.clone()` that returns another usable instance of the Rc. But I don't want to keep it for later, because cpu doesn't need to return the memory, but gba. No problem for now, just a reminder for later.

Reply:
Once you merge #111 and I rebase this `get_memory` fn will probably go away